infra/modules/psoxy-package/build.sh

```diff
@@ -7,14 +7,31 @@ set -e
 TERRAFORM_CONFIG_PATH=`pwd`
 LOG_FILE=${TERRAFORM_CONFIG_PATH}/psoxy-package.`date +%Y%m%d'T'%H%M%S`.log
 
-cd $1/gateway-core
-mvn package install > ${LOG_FILE} 2>&1
+FOLDERS=("gateway-core" "core" "impl/$2")
 
-cd $1/core
-mvn package install > ${LOG_FILE} 2>&1
+touch ${LOG_FILE}
 
-cd $1/impl/$2
-mvn package >> ${LOG_FILE} 2>&1
+cd $1
+for index in "${!FOLDERS[@]}"
```

Comment:
this can be reverted if you want, still needs some tweaks, but basically allows for faster turnaround when testing lambdas but changes only affect terraform. It avoids regenerating the jars.
Pending: regenerate all if any is stale.

Reply:
yeah, this is definitely a pain point that's worth solving; but perhaps decouple from this SSM stuff, as should be totally separate but we need to be absolutely certain it's working correctly before release

Reply:
reverted, I'll work on a new branch with what I had so far